config.json

```diff
@@ -0,0 +1,13 @@
+{
+	"database_settings": {
+		"username":"",
+		"password":"",
+		"host":"",
+		"port":"",
+		"database":"",
+		"sslmode":""
+	},
+	"logger_settings": {
+		"dump_request":""
+	}
+}
```

Comment:
This file might be described in a readme but probably should not be committed to git.

Reply:
Ok, I'll add it to gitignore, and add issues for updating the readme and adding config options